src/content/docs/fr/guides/migrate-to-astro/from-wordpress.mdx

```diff
@@ -12,7 +12,7 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 
 :::tip
-Vous pouvez [utiliser WordPress comme headless CMS pour votre projet Astro](/fr/guides/cms/wordpress/). Suivez notre guide pour utiliser votre contenu WordPress existant dans un nouveau projet Astro.
+Vous pouvez [utiliser WordPress comme CMS headless (ou « sans-tête ») pour votre projet Astro](/fr/guides/cms/wordpress/). Suivez notre guide pour utiliser votre contenu WordPress existant dans un nouveau projet Astro.
```

Comment:
I rephrased it slightly because we use both terms in French. (and that explains why, to reduce repetition, I use both terms in community resources)